packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -97,6 +103,10 @@ function _domBreadcrumb(dom: BreadcrumbsOptions['dom']): (handlerData: { [key: s
   function _innerDomBreadcrumb(handlerData: { [key: string]: any }): void {
     let target;
     let keyAttrs = typeof dom === 'object' ? dom.serializeAttribute : undefined;
+    const maxStringLength =
```

Comment:
Maybe you could add something like:

```js
if (__DEBUG_BUILD__ && typeof dom.maxStringLength === 'number' && dom.maxStringLength > 1024) {
  logger.warn(`\`dom.maxStringLength\` cannot exceed 1024, but a value of ${dom.maxStringLength} was configured. Sentry will use 1024 instead.`);
}
```

I think that would be a valuable warning to have, instead of basically silently capping this.

Reply:
Good call. Updated!